### DISCUSSION TITLE
Show custom command result as element in statusline.

### DISCUSSION THREAD

**Discussion by Tadjaur** [helix-editor/helix#8381] [Ideas] [answered]
Hi, 
I'm pretty new here and i commonly make a mistake of forgiving the "caps lock" key on. 
I would like to show a ***CAPS*** tag in the `statusline`, when the caps lock is on.
I'm on Linux machine and i already have my custom command: `xset q | grep "Caps Lock:   on" > /dev/null && echo 'CAPS'` but I don't know how to show that in the `statusline`.
I want to know if it is possible?
If not, is there another side where I can easily show that?

**Comment by the-mikedavis** [comment, marked as answer]
Custom statusline elements could be supported by the plugin system when that is added. There isn't currently a way to make them.
